mungegithub/mungers/submit-queue.go

```diff
@@ -499,6 +499,7 @@ func (sq *SubmitQueue) internalInitialize(config *github.Config, features *featu
 		if sq.BatchEnabled {
 			http.Handle("/batch", gziphandler.GzipHandler(http.HandlerFunc(sq.serveBatch)))
 		}
+		// this endpoint is useless without access to prow
 		if sq.ProwURL != "" {
 			http.Handle("/ci-status", gziphandler.GzipHandler(http.HandlerFunc(sq.serveCIStatus)))
```

Comment:
I think serving both of those paths regardless of whether they are enabled is a good idea for the sake of other services that might have dependencies on the submit queue. Its probably better for those services to receive empty lists of results rather than 404s. That change might belong in a separate PR though so I'll 
/lgtm
this one.